packages/components/src/core/Panel/index.tsx

```diff
@@ -71,6 +78,21 @@ const Panel = React.forwardRef<HTMLDivElement, PanelProps>((props, ref) => {
       anchor={drawerAnchor}
       variant={drawerVariant}
       width={drawerWidth}
+      ModalProps={{
+        ...ModalProps,
+        // (masoudmanson): Set disableEnforceFocus: true to prevent the Drawer from forcing
```

Comment:
really great comments thank you so much for sharing the knowledge 🙇

Reply:
Thanks for the review 😍🎉😬